templates/common/.azdo/pipelines/bicep/azure-dev.yml

```diff
@@ -8,6 +8,9 @@ pool:
 container: mcr.microsoft.com/azure-dev-cli-apps:latest
 
 steps:
+  - pwsh: |
+      azd config set auth.useAzCliAuth "true"
+    displayName: Configure AZD to Use AZ CLI Authentication.
```

Comment:
Is this to test coverage of using `az`? Do we have a follow-up issue to flip this? 

Or are there technical limitations?

Reply:
The issue is that all the commands in this pipeline use the `AzureCLI@2` task to invoke a script while signed into a specific azure connection (if you expand the diff see what's going on) and we don't support that yet.  This was the most obvious fix - I'll file a follow up issue so we can support this natively long term.

Reply:
#1126

Reply:
I see. It uses a service connection, which requires us to expand that if we wanted to support it.